client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -26,6 +28,6 @@ export default [
 	PublicizeBlock,
 	SimplePaymentsBlock,
 	...( isEnabled( 'jetpack/blocks/beta' )
-		? [ RelatedPostsBlock, TiledGalleryBlock, VRBlock ]
+		? [ RelatedPostsBlock, TiledGalleryBlock, VRBlock, EmailSubscribeBlock ]
```

Comment:
Let's keep this list sorted alphabetically.

Reply:
Done!